src/main/java/rotateit/service/github/label/OnLabelEventToPersistListener.java

```diff
@@ -5,7 +5,7 @@
 
 public class OnLabelEventToPersistListener {
 
-    @EventListener
+    @EventListener(condition = "#labelEvent.pullRequestInOpenState")
```

Comment:
We have 2 listeners that are listening on the same conditions, please remove one of them.